.github/workflows/build.yaml

```diff
@@ -0,0 +1,37 @@
+name: Build
+
+on:
+  push:
+    branches:
+    - main
+
+  pull_request:
```

Comment:
I think it should be good enough to only publish to IPFS on push (might be a lot of content to push to IPFS if it's uploaded for every PR).

Reply:
agree, will fix

Reply:
fixed